### ISSUE TITLE
Training segnet with city Scapes Data set

### ISSUE THREAD

**Issue by saviogeorge** [TimoSaemann/caffe-segnet-cudnn5#16]
Hi,
I was trying to train the segnet using the city scapes data set However i faced issues.
like
![out of memory](https://user-images.githubusercontent.com/20711873/28366923-f91a175a-6c8e-11e7-8aec-732d15f52937.png)

for which I thought I will resize the image to 480x360
Then the next issue was
![annotations_problem](https://user-images.githubusercontent.com/20711873/28366949-12d0d62a-6c8f-11e7-8f47-a48b1be2926c.png)

which is yet to be solved. is it because of the annotations provided by city scapes dataset has a problem with segnet if so how can i solve it.

In short i would like to know how to train the segnet using the city scapes data set.

Thank You!
savio
